sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/implementation/ChangeFeedContextClientImpl.java

```diff
@@ -40,7 +43,7 @@
 public class ChangeFeedContextClientImpl implements ChangeFeedContextClient {
     private final AsyncDocumentClient documentClient;
     private final CosmosAsyncContainer cosmosContainer;
-    private final Scheduler rxScheduler;
+    private Scheduler rxScheduler;
```

Comment:
Just to be clear, `rx` means or meant RxJava, can we please call it `reactorScheduler` or just `scheduler` ?

Reply:
yep, I can do the rename; I think it was left over from the initial implementation